code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -513,6 +513,29 @@
 			return FALSE
 	return ..()
 
+// --------------------- Standard
+/obj/item/robot_model/standard
+	name = "Standard"
+	basic_modules = list(
+		/obj/item/assembly/flash/cyborg,
+		/obj/item/reagent_containers/borghypo/epi,
+		/obj/item/healthanalyzer,
+		/obj/item/weldingtool/largetank/cyborg,
+		/obj/item/wrench/cyborg,
+		/obj/item/crowbar/cyborg,
+		/obj/item/stack/sheet/metal,
+		/obj/item/stack/rods/cyborg,
+		/obj/item/stack/tile/plasteel,
+		/obj/item/extinguisher,
+		/obj/item/pickaxe/cyborg,
+		/obj/item/t_scanner/adv_mining_scanner,
+		/obj/item/restraints/handcuffs/cable/zipties,
```

Comment:
WOOOOOOOOOOOO!! YEAAAAAAH BABY THATS WHAT IVE BEEN WAITING FOR THATS WHAT ITS ALL ABOUT WOOOOOOOOO